### PR TITLE
Random Pull

### PR DESCRIPTION
**Pull request by other-rd95**
Check this

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,15 @@
 #Graph
+
+Simple Graph Program which implements the following:
+
```
Comment (dan13ram) [inline README.md:4]: This looks good

File: README.md [hunk 2]
```diff
@@ -1 +1,15 @@
...
+1. Breadth First Search (BFS)
+2. Kruskal's algorithm for Minimum Spanning Tree
+3. Prim's algorithm for Minimum Spanning Tree
+4. Dijkstra's Shortest Path Algorithm
+
+To Compile:
+g++ -std=c++11 main.cpp -o simple_graph
```
Comment (dan13ram) [inline README.md:11]: Do you think you can change the code to compile with normall g++ ?

**Review by dan13ram — APPROVED** [review]
LGTME

File: README.md (hunk 2 above)
Comment (dan13ram) [reply README.md:11]: Do this fast

Comment (dan13ram) [reply README.md:11]: Come on bro

**Review by dan13ram — CHANGES_REQUESTED** [review]

**Comment by other-rd95** [comment]
DUde I changed it already. Check properly!

**Review by dan13ram — APPROVED** [review]

**Pull request closed without merging**